graphql/image_test.go

```diff
@@ -65,6 +65,41 @@ func TestToolchains(t *testing.T) {
 	assert.Equal(t, testToolchain, utility.FromStringPtr(res[0].Name))
 }
 
+func TestEvents(t *testing.T) {
+	config := New("/graphql")
+	ctx := getContext(t)
+	testConfig := testutil.TestConfig()
+	testutil.ConfigureIntegrationTest(t, testConfig, "TestEvents")
+	require.NoError(t, testConfig.RuntimeEnvironments.Set(ctx))
+
+	// Returns the correct number of events according to the limit.
+	imageID := "ubuntu2204"
+	image1 := model.APIImage{
+		ID: &imageID,
+	}
+	res, err := config.Resolvers.Image().Events(ctx, &image1, 5, 0)
+	require.NoError(t, err)
+	assert.Len(t, res, 5)
+
+	// Does not return the same events in different pages.
+	allAMIAfter := map[string]struct{}{}
+	for _, event := range res {
+		allAMIAfter[utility.FromStringPtr(event.AMIAfter)] = struct{}{}
+	}
```

Comment:
I think it's not exactly clear what's going on here and that it would be simpler to use `map[string]bool{}`. 

We should probably get a more explicit name like `amiSet` to help the reader understand what's going on:

```suggestion
	amiSet := map[string]bool{}
```

Reply:
rewrote test according to Bynn's suggestion